docs/metrics.rst

```diff
@@ -18,10 +18,17 @@
 Metrics
 =======
 
-Airflow can be set up to send metrics to `StatsD <https://github.com/etsy/statsd>`__:
+Airflow can be set up to send metrics to `StatsD <https://github.com/etsy/statsd>`__.
+
+Setup
+-----
+
+First you must install statsd requirement:
+
+.. code-block:: bash
+
+   pip install apache-airflow[statsd]
```

Comment:
```suggestion
   pip install `apache-airflow[statsd]`
```
Reference: https://github.com/apache/airflow/pull/4966

Reply:
Actually, shouldn't it be `pip install 'apache-airflow[statsd]'`